src/plots/cartesian/axes.js

```diff
@@ -1905,7 +1905,7 @@ axes.doTicks = function(gd, axid, skipTitle) {
     var axLetter = axid.charAt(0),
         counterLetter = axes.counterLetter(axid),
         vals = axes.calcTicks(ax),
-        datafn = function(d) { return [d.text, d.x, ax.mirror].join('_'); },
+        datafn = function(d) { return [d.text, d.x, ax.mirror, d.font, d.fontSize, d.fontColor].join('_'); },
```

Comment:
Cartesian axes use a special [`ticks`](https://github.com/plotly/plotly.js/blob/599dff93f9923e7ef4d6446de734cfb4eb39d663/src/plot_api/subroutines.js#L484) update pathway that.

This here gets `Axes.doTicks` to update properly. I'm pretty sure some axis style updates are currently broken for ternary subplots. I'll lock those down in tests soon.

Reply:
:lock: in https://github.com/plotly/plotly.js/pull/2200/commits/7b0f718d8961794fc395ef249cdb6462c25914b9